app/main/presenters/search_results.py

```diff
@@ -5,28 +5,28 @@ class SearchResults(object):
     """Provides access to the search results information"""
 
     def __init__(self, response, lots_by_slug):
-        self.search_results = response.get('documents') or response.get('services')
+        self.search_results = response['documents']
```

Comment:
is the response object now an array?

Reply:
It's a dict, with an array of results, which are now behind the `documents` key.